pkg/kubectl/cmd/create_job.go

```diff
@@ -37,110 +34,96 @@ var (
 		Create a job with the specified name.`))
 
 	jobExample = templates.Examples(i18n.T(`
+		# Create a job
+		kubectl create job my-job --image=busybox
+
 		# Create a job from a CronJob named "a-cronjob"
 		kubectl create job --from=cronjob/a-cronjob`))
 )
 
-type CreateJobOptions struct {
```

Comment:
Keep this struct.  Think of this options struct is your entry point for unit testing.  It provides a spot to add your mocks without trying to plumb an entire command.